LCLSGeneral/LCLSGeneral/POUs/Hardware/FB_LED.TcPOU

```diff
@@ -21,6 +21,12 @@
     2022-5-03 Maarten Thomas-Bosum
 *)
 VAR_INPUT
+	{attribute 'pytmc' := '
+        pv: NAME
+        io: io 
```

Comment:
I'd recommend adding something like this to the pragma:
```
field: DESC Descriptive name for the LED
autosave_pass0: VAL DESC
```

Autosave reference: https://pcdshub.github.io/pytmc/v2.9.1/pragma_usage.html#autosave

Reply:
```
{attribute 'pytmc' := '
        pv: NAME
        io: io 
	field: DESC Descriptive name for the LED
        autosave_pass0: VAL DESC
    '}
	ledName : STRING;
```

@klauer So this is what you had in mind?

Reply:
(I had this tab open and forgot to reply, sorry @Mbosum )

Yes, that's what I was suggesting 👍

Reply:
Great, thanks for answering!